specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/billingSavingsPlan.json

```diff
@@ -832,6 +843,9 @@
         },
         "extendedStatusInfo": {
           "$ref": "#/definitions/ExtendedStatusInfo"
+        },
+        "productCode": {
```

Comment:
ProductCode we should keep _inside_ properties for now, because this isn't a well-known ARM property today (while tags *is*).

Reply:
@wilcobmsft `productCode` is within the the properties `SavingsPlanOrderModelProperties` (line 792 to line 85)